### PR TITLE
Updated example with a has_many through association

### PR DESCRIPTION
**Pull request by elbow-jason**
+ added City model
+ added Country model
+ added migration file
+ updated example's README.md

This is a work in progress. Any/All feedback is welcome.

### CODE REVIEW

File: lib/ecto/schema.ex [hunk 1]
```diff
@@ -483,6 +483,7 @@ defmodule Ecto.Schema do
           # and the `:author` refers to the `belongs_to :author`
           # of the Comment module's schema (the module below).
           # (see the description below for more details)
+
```
Comment (josevalim) [inline lib/ecto/schema.ex]: No need to add this space here. :)

File: examples/simple/priv/repo/migrations/20150710205000_create_city_and_country.exs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+defmodule Simple.Repo.Migrations.CreateCityAndCountry do
```
Comment (josevalim) [inline examples/simple/priv/repo/migrations/20150710205000_create_city_and_country.exs]: No need to add this migration, just change the original one please. Nobody is running this in production. :D

File: examples/simple/lib/simple.ex [hunk 3]
```diff
@@ -17,21 +17,62 @@ defmodule Weather do
...
+
+  Without `preload: weather` the :weather field for all loaded Countries
+  would be an `Ecto.Association.NotLoaded` struct.
+  """
+  def countries_with_weather_query do
+    query = from c in Country,
+         preload: :weather
+    Simple.Repo.all(query)
+  end
+
```
Comment (josevalim) [inline examples/simple/lib/simple.ex]: Please remove extra new lines :scissors:  :)

**Comment by josevalim** [comment]
Beautiful, I have added some comments.

**Comment by josevalim** [comment]
Merged manually, thank you!

**Comment by elbow-jason** [comment]
Awesome! Thanks!

**Pull request closed without merging**